src/components/entity/state-badge.ts

```diff
@@ -111,15 +113,38 @@ class StateBadge extends LitElement {
         transition: color 0.3s ease-in-out, filter 0.3s ease-in-out;
       }
 
-      /* Color the icon if light or sun is on */
-      ha-icon[data-domain="light"][data-state="on"],
-      ha-icon[data-domain="switch"][data-state="on"],
+      ha-icon[data-domain="alarm_control_panel"][data-state="disarmed"],
+      ha-icon[data-domain="alert"][data-state="on"],
+      ha-icon[data-domain="automation"][data-state="on"],
       ha-icon[data-domain="binary_sensor"][data-state="on"],
+      ha-icon[data-domain="calendar"][data-state="on"],
+      ha-icon[data-domain="camera"][data-state="streaming"],
+      ha-icon[data-domain="cover"][data-state="open"],
       ha-icon[data-domain="fan"][data-state="on"],
-      ha-icon[data-domain="sun"][data-state="above_horizon"] {
+      ha-icon[data-domain="light"][data-state="on"],
+      ha-icon[data-domain="input_boolean"][data-state="on"],
+      ha-icon[data-domain="lock"][data-state="unlocked"],
+      ha-icon[data-domain="media_player"][data-state="paused"],
+      ha-icon[data-domain="media_player"][data-state="playing"],
+      ha-icon[data-domain="plant"][data-state="problem"],
```

Comment:
I find it not ideal that "needs attention" is the strategy. I think maybe "doing its thing" is a bette rone. A camera is streaming, the sun is above the horizon, a timer is active. Now there is all of a sudden a problem state in it. Same with Z-Wave. I think that we should not highlight problems, because then people for sure will request that we color them differently !

Reply:
it we add per entity theming, as you suggested, it would be a moot-point on their part.

Reply:
Right. But our default UI should still make sense ! If we render errors in red, a dead or problem state should not be yellow ?

Reply:
I've extracted these
```
  ha-icon[data-domain="plant"][data-state="problem"],
  ha-icon[data-domain="zwave"][data-state="dead"] {
    color: var(--error-state-color, #db4437);
  }
```
Is this acceptable?